src/combinators/converge.js

```diff
@@ -0,0 +1,17 @@
+/** @license ISC License (c) copyright 2016 original and current authors */
+/** @author Matt Ross (amsross) */
+
+const curry = require('../helpers/curry')
```

Comment:
I am sorry that I also missed this in my original review:
both `curry` and `isFunction` should come from `require('../core/curry')` and `require('../core/isFunction')`.
Once we move to a mono-repo, `@crocksjs/core` will have to be installed to keep the other packages light.

Again, I am sorry I missed this.